### DISCUSSION TITLE
How to create a new nav tab in a model template.

### DISCUSSION THREAD

**Discussion by renatoalmeidaoliveira** [netbox-community/netbox#7305] [Q&A] [answered]
There're a way to add a custom nav tab with plugins?

**Comment by jeremystretch** [comment, marked as answer]
If you're referring to the general navigation menu in NetBox, yes, it's supported: https://netbox.readthedocs.io/en/stable/plugins/development/#navigation-menu-items

**Reply by renatoalmeidaoliveira** [reply]
It's those tabs ( highlight in green) I would like add a custom one
![image](https://user-images.githubusercontent.com/16046203/134008620-8ac5f910-1b2b-4fb9-9887-b8ee2ed6c242.png)

**Reply by jeremystretch** [reply]
Sure, you can do whatever you'd like within your own custom template. However, you can not add tabs to models outside your own plugin.

**Reply by renatoalmeidaoliveira** [reply]
Thanks, it's that what I whish to add a new tab to an existing model.
Like extend the Devices models to include a tab from mine plugin.

**Reply by jeremystretch** [reply]
No; the tabs are part of the core templates. However, you can easily add custom links for any model.

**Reply by renatoalmeidaoliveira** [reply]
Too bad... It's a design choice? or something that could be added to the core templates like a:
` {% plugin_navs object %}`
inside the [generic object](https://github.com/netbox-community/netbox/blob/develop/netbox/templates/generic/object.html) template

**Reply by jeremystretch** [reply]
You're welcome to submit a feature request for it.
